package.json

```diff
@@ -41,14 +41,45 @@
+    "all": "npm run dist && npm run dist/rdflib.js && npm run dist/rdflib.min.js",
+    "lib": "babel src -d lib",
+    "detach": "git checkout origin/master &&  git reset--hard HEAD",
+    "clean": "rimraf dist/* lib/*",
+    "cleantest": "rimraf tests/serialize/,*",
+    "status": "@git branch -v && @git status -s",
+    "minify": "npm run dist/rdflib.min.js",
+    "size": "node-line-counter .",
+    "gh-pages": "npm run detach && git branch - D gh - pages ||: && git checkout - b gh - pages && npm run all && git add - f dist /*.js src/*.js *.js && git commit -m 'gh-pages: update to latest' && git push -f origin gh-pages && git checkout master",
+    "prepublish": "npm run lib",
+    "serialize-tests": "npm run serialize-test-1 && npm run serialize-test-2 && npm run serialize-test-3 && npm run serialize-test-4 && npm run serialize-test-5 && npm run serialize-test-6 && npm run serialize-test-7 && npm run serialize-test-10 && npm run serialize-test-11 && npm run serialize-test-12 && npm run serialize-test-13",
+    "serialize-test-1": "cd tests/serialize && node ./data.js -in=t1.ttl -format=application/rdf+xml -out=,t1.xml && fs-grep http://www.w3.org/2001/XMLSchema#integer ,t1.xml",
+    "serialize-test-2": "cd tests/serialize && node ./data.js -in=t2.ttl -format=application/rdf+xml  -out=,t2.xml && jsdiff ,t2.xml t2-ref.xml",
+    "serialize-test-3": "cd tests/serialize && node ./data.js -in=t3.ttl -format=application/rdf+xml  -out=,t3.xml && jsdiff ,t3.xml t3-ref.xml",
+    "serialize-test-4": "cd tests/serialize && node ./data.js -in=t3.ttl -out=,t4.ttl && jsdiff ,t4.ttl t4-ref.ttl",
+    "serialize-test-5": "cd tests/serialize && node ./data.js -in=t5.n3 -format=text/turtle -out=,t5.ttl && jsdiff ,t5.ttl t5-ref.ttl",
+    "serialize-test-6": "cd tests/serialize && node ./data.js -in=t5.n3 -format=text/n3 -out=,t6.n3 && jsdiff ,t6.n3 t6-ref.n3",
+    "serialize-test-7": "cd tests/serialize && node ./data.js -in=t7.n3 -format=application/n-triples -out=,t7.nt && jsdiff ,t7.nt t7-ref.nt",
+    "serialize-test-8": "cd tests/serialize && node ./data.js -in=t5.n3  -format=application/n-quads -dump=,t8.nq && jsdiff ,t8.nq t8-ref.nq",
+    "serialize-test-9": "cd tests/serialize && node ./data.js -in=t7.n3 -format=application/ld+json -out=,t9.json && jsdiff ,t9.json t9-ref.json",
+    "serialize-test-10": "cd tests/serialize && node ./data.js -in=details.ttl -format=text/turtle -out=,t10.ttl && jsdiff ,t10.ttl t10-ref.ttl",
+    "serialize-test-11": "cd tests/serialize && node ./data.js -in=structures.n3 -format=text/turtle -out=,structures.ttl && jsdiff ,structures.ttl t12-ref.ttl",
+    "serialize-test-12": "cd tests/serialize && node ./data.js -in=t5.n3  -format=application/n-quads -dump=,t8.nq && jsdiff ,t8.nq t8-ref.nq",
```

Comment:
This is the wrong script. Check out the source of test 12.

Also this test fails, but it's not properly failing the travis build. It seems like `jsdiff` doesn't return a failure exit code like `diff` does. Can you look into making `jsdiff` fail for files that don't match?